### PR TITLE
updating caption

### PR DESCRIPTION
**Pull request by chambersDon**

### CODE REVIEW

**Comment by chambersDon** [comment]
test

**Review by lonturner — APPROVED** [review]

File: _slides/06-chambersDon.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+My New Caption
```
Comment (lonturner) [inline _slides/06-chambersDon.md:6]: Well done!

**Review by lonturner — CHANGES_REQUESTED** [review]
Some change request...

**Review by lonturner — APPROVED** [review]
Good to go.

**Pull request merged**